coralproject/talk-fe
# Updating embedlink to be customizable

## What does this PR do?

`coral-admin` no longer assumes that the embed script will be located
on the same server, instead accepting a config variable which states
where `coral-embed-stream` is hosted. This prevents possible confusion when two (potentially out of sync) versions of `coral-embed-stream` are being hosted on staging.

## How do I test this PR?

1. Run `coral-admin` according to the instructions in the Readme.
2. Visit `/embedlink`
3. The embeddable link should read:
```<div id='coralStreamEmbed'></div><script type='text/javascript' src='http://pym.nprapps.org/pym.v1.min.js'></script><script>var pymParent = new pym.Parent('coralStreamEmbed', ‘[CONFIGURABLE_HOST]/embedScript/index.html', {});</script>``` 

Where [CONFIGURABLE_HOST] is the `streamEmbedUrl` value listed in `/coral-admin/config.json`.

## Note
@wyattjoh will need to update the staging build process to set this config variable.
